apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.v3.$projectParam.alerts/route.tsx

```diff
@@ -187,14 +187,12 @@ export default function Page() {
           </div>
           <Table>
             <TableHeader>
-              <TableRow>
-                <TableHeaderCell>Name</TableHeaderCell>
-                <TableHeaderCell>Alert types</TableHeaderCell>
-                <TableHeaderCell>Environments</TableHeaderCell>
-                <TableHeaderCell>Channel</TableHeaderCell>
-                <TableHeaderCell>Enabled</TableHeaderCell>
-                <TableHeaderCell hiddenLabel>Actions</TableHeaderCell>
-              </TableRow>
+              <TableHeaderCell>Name</TableHeaderCell>
+              <TableHeaderCell>Alert types</TableHeaderCell>
+              <TableHeaderCell>Environments</TableHeaderCell>
+              <TableHeaderCell>Channel</TableHeaderCell>
+              <TableHeaderCell>Enabled</TableHeaderCell>
+              <TableHeaderCell hiddenLabel>Actions</TableHeaderCell>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"